playbooks/06-deploy-architecture.yml

```diff
@@ -37,7 +37,14 @@
         - not _nova_key_file.stat.exists
       community.crypto.openssh_keypair:
         comment: "nova migration"
-        path: "{{ _nova_key_file.stat.path }}"
+        path: >-
```

Comment:
can't we make a block around those tasks (with the `ansible.builtin.stat` and a `vars` in order to avoid the repetition? Since `ansible.builtin.stat` seems to *remove* the `path` attribute if the file doesn't exist, that's the right way to avoid repetition all the way...

Thoughts?

```YAML
- name: Manage nova_migration key
  vars:
    _sshkey: "{{ <build path> }}"
  block:
    - name: Look for nova migration keypair file
    - name: Create nova migration keypair if does not exists
    - name: Generate needed facts out of local files

Reply:
Thank you. Yes, it make's sense.

Let me update it in my incoming commit.